pkgs/tools/system/tre/default.nix

```
@@ -0,0 +1,24 @@
+{ rustPlatform, fetchFromGitHub, stdenv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tre-${version}";
```

```suggestion
  pname = "tre";
```

The name is contrusted of ```${pname}-${version}``` so you should use just the name itself.